src/js/views/views/AwardsRecords.js

```diff
@@ -0,0 +1,46 @@
+const React = require('react');
+const getCols = require('../../util/getCols');
+const helpers = require('../../util/helpers');
+const bbgmViewReact = require('../../util/bbgmViewReact');
+
+const {DataTable, Dropdown, NewWindowLink} = require('../components/index');
+
+const AwardsRecords = ({awardsRecords, playerCount, awardTypeVal}) => {
+    bbgmViewReact.title('Awards Records');
+    const cols = getCols('Name', 'Count', 'Year', 'Last', 'Retired', 'HOF');
+
+    const rows = awardsRecords ? awardsRecords.map(a => {
```

Comment:
Solve this by giving awardsRecords a default value, like line 8:

    const AwardsRecords = ({awardsRecords = [], playerCount, awardTypeVal}) => {

That's what I do elsewhere. Later on I'm going to go back and ensure that correct types are actually passed to these components all the time.